packages/trader/src/Modules/Contract/Containers/contract-replay.jsx

```diff
@@ -11,7 +11,7 @@ import {
     FadeWrapper,
 } from '@deriv/components';
 import { isDesktop, isMobile, isEmptyObject } from '@deriv/shared';
-
+import { getPlatformRedirect } from '@deriv/shared';
```

Comment:
```suggestion
```

Reply:
@zaki-binary thank you for noticing, just updated it.